pom.xml

```diff
@@ -137,10 +137,6 @@
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
                     <excludes>
-                        <exclude>
-                            <groupId>org.projectlombok</groupId>
```

Comment:
Does it throw any errors once you take this off?

Reply:
no